docs/ref/request-response.txt

```diff
@@ -167,7 +167,7 @@ All attributes should be considered read-only, unless stated otherwise.
     underscores in WSGI environment variables. It matches the behavior of
     Web servers like Nginx and Apache 2.4+.
 
-    :attr:`HttpRequest.headers` is a simpler way to access all HTTP-prefixed
+    :attr:`HttpRequest.headers` is another way to access all HTTP-prefixed
```

Comment:
I think this can be reverted, it's simpler and it's intended to be used because it is simpler